pkg/shell/cockpit-util.js

```diff
@@ -307,4 +307,29 @@ function hostname_for_display(iface) {
         return window.location.hostname;
 }
 
+/* - name = shell.update_privileged_ui
+ *
+ * Update the disabled status and tooltip for all
+ * elements with the given css class.
+ */
+shell.update_privileged_ui = function update_privileged_ui(perm, css_class, denied_message) {
```

Comment:
In general, if code is going to be reused between components (think pages) we need to figure out how to get it into the base package. Usually that means getting it into the cockpit namespace, and making it public API. Should we brainstorm how to do that for this code? Is it going to be actively shared between components?